addons/hr_holidays/data/hr_holidays_demo.xml

```diff
@@ -438,5 +438,14 @@
         <field name="state">confirm</field>
     </record>
 
+    <!-- Public time off -->
+    <record id="resource_public_time_off_1" model="resource.calendar.leaves">
+        <field name="name">Public Time Off</field>
+        <field name="company_id" ref="base.main_company"/>
+        <field name="calendar_id" ref="resource.resource_calendar_std"/>
+        <field name="date_from" eval="(datetime.today() + relativedelta(months=+1)).strftime('%Y-%m-%d 07:00:00')"></field>
+        <field name="date_to" eval="(datetime.today() + relativedelta(months=+1)).strftime('%Y-%m-%d 16:00:00')"></field>
+    </record>
```

Comment:
If it is just for planning app, then we could add this demo data in planning_holidays module for instance. What do you think? 🙂